xml/Microsoft.Build.BuildEngine/LocalNode.xml

```diff
@@ -19,12 +19,12 @@
   </Docs>
   <Members>
     <Member MemberName="StartLocalNodeServer">
-      <MemberSignature Language="C#" Value="public static void StartLocalNodeServer (int nodeNumber);" />
-      <MemberSignature Language="ILAsm" Value=".method public static hidebysig void StartLocalNodeServer(int32 nodeNumber) cil managed" />
-      <MemberSignature Language="DocId" Value="M:Microsoft.Build.BuildEngine.LocalNode.StartLocalNodeServer(System.Int32)" />
-      <MemberSignature Language="VB.NET" Value="Public Shared Sub StartLocalNodeServer (nodeNumber As Integer)" />
-      <MemberSignature Language="C++ CLI" Value="public:&#xA; static void StartLocalNodeServer(int nodeNumber);" />
-      <MemberSignature Language="F#" Value="static member StartLocalNodeServer : int -&gt; unit" Usage="Microsoft.Build.BuildEngine.LocalNode.StartLocalNodeServer nodeNumber" />
+      <MemberSignature Language="C#" Value="public static void StartLocalNodeServer (int nodeNumber);" FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
```

Comment:
why was this extra info added?

Reply:
API doesn't exist in all monikers ... this will be more common to see the full subset of moniker availability in APIs (though it may be optimized away in the future if possible). But for now, not a cause for concern unless the list is wrong